geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreFactory.scala

```diff
@@ -224,7 +222,7 @@ object HBaseDataStoreFactory extends LazyLogging {
         UserGroupInformation.loginUserFromKeytab(conf.get(HBaseGeoMesaPrincipal), conf.get(HBaseGeoMesaKeyTab))
 
       case _ =>
-        logger.debug(s"Hadoop is not configured to use Kerberos.  Setting 'hbase.security.authentication' was set to $auth.")
```

Comment:
haha, i guess that actually did make sense before, i just read 'setting' as a verb instead of a noun